### DISCUSSION TITLE
Duda al mostrar nombre en el HTML Pokedex

### DISCUSSION THREAD

**Discussion by LuisMelendezFuentes** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#585] [Q&A]
Hola colegas,

estoy realizando mi pokedex pero no encuentro la forma de como hacer para que se muestre el nombre del pokemon que se busco en en la pantalla, si aparece en consola pero no logro que se muestre en el HTML, les muestro mi código por si alguien me puede orientar se lo agradecería infinitamente,
![HTML](https://user-images.githubusercontent.com/99149751/162278344-d24fa76d-2605-4b80-aaf7-7be89ec193d1.png)
![JS](https://user-images.githubusercontent.com/99149751/162278359-b65bd062-abaf-4d6e-a9e6-b3285c982690.png)

**Comment by ismagician** [comment]
Hola, no te funciona porque no estas asignando el valor del Pokemon al elemento HTML agrega 
```const pokeNombre = document.getElementById("pokeNombre");```
y para pasarle el nombre
```pokeNombre.textContent=data.forms[0].name; ```

**Reply by LuisMelendezFuentes** [reply]
Muchas gracias, lo intento como me comentas pero aun no e logrado mostrarlo

**Reply by ismagician** [reply]
Puedes subir una captura de como lo estas haciendo por fa. 
Te muestro un ejemplo 
![imagen](https://user-images.githubusercontent.com/92187317/162490590-39b1cf2c-de18-4d06-bcf2-3ddfb5d49b35.png)
![imagen](https://user-images.githubusercontent.com/92187317/162490710-b83f9154-331c-49aa-a22d-5a210ad8fa6c.png)

**Reply by LuisMelendezFuentes** [reply]
![image](https://user-images.githubusercontent.com/99149751/162495101-9e344c7d-d41e-4394-a227-72ff5b57223b.png)
![image](https://user-images.githubusercontent.com/99149751/162495112-3336bfef-4ea4-42e1-9ff4-03f69fb0d866.png)
![image](https://user-images.githubusercontent.com/99149751/162495124-3872251d-809c-4232-be2b-df565a9acf18.png)
![image](https://user-images.githubusercontent.com/99149751/162495138-38f348f3-c0ca-44e3-af8c-068ec295eca9.png)

Así lo tengo de momento

**Reply by ismagician** [reply]
No le estás pasando nada a la función pokeNombre y tampoco la estas llamando. 
![imagen](https://user-images.githubusercontent.com/92187317/162503929-496f0316-06d1-45c8-8138-fb9f38afe4e6.png)
Para que solo llames una función puedes crear un json y lo pasas, quedaría algo así
![imagen](https://user-images.githubusercontent.com/92187317/162504709-e2f160be-67ca-4a76-9b50-50736165f261.png)
